src/lib/plugin.js

```diff
@@ -808,9 +808,8 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   _requestRetry (requestOptions, retryOptions) {
-    return requestRetry(Object.assign({
-      credentials: this.credentials
-    }, requestOptions), retryOptions)
+    return requestRetry(Object.assign(requestCredentials(this.credentials), requestOptions),
```

Comment:
`requestCredentials(...)` returns a new JSON object everytime it is called, so it should be not necessary to create an empty JSON object first, no?